scripts/send-bundles-sizes-differences.js

```diff
@@ -0,0 +1,181 @@
+const { execSync } = require('child_process')
+const { runMain, timeout } = require('./lib/execution-utils')
+const { getOrg2ApiKey, getGithubAccessToken, getOrg2AppKey } = require('./lib/secrets')
+
+const header = 'Bundles Sizes Evolution'
+const baseBranch = 'main'
+const localBranch = process.env.CI_COMMIT_REF_NAME
+const githubAuthToken = execSync('authanywhere').toString().split(' ')[2].trim()
+const GH_TOKEN = getGithubAccessToken()
+
+const getLastCommonCommit = (branch) => {
+  try {
+    execSync('git fetch origin')
+    const command = `git merge-base origin/${branch} HEAD`
+    const commit = execSync(command).toString().trim()
+    const shortCommand = `git rev-parse --short=8 ${commit}`
+    return execSync(shortCommand).toString().trim()
+  } catch (error) {
+    throw new Error('Failed to get last common commit', { cause: error })
+  }
+}
+
+async function getPRs(branch) {
+  const response = await fetch(`https://api.github.com/repos/DataDog/browser-sdk/pulls?head=DataDog:${branch}`, {
+    method: 'GET',
+    headers: {
+      Authorization: `token ${GH_TOKEN}`,
+    },
+  })
+  if (!response.ok) {
+    throw new Error(`HTTP Error Response: ${response.status} ${response.statusText}`)
+  }
+  return response.body ? response.json() : null
+}
+
+function createMessage(difference, resultsBaseQuery, resultsLocalQuery) {
+  let message = '| Bundle | Base Size | Current Size | 𝚫% |\n| --- | --- | --- | --- |\n'
+
+  difference.forEach((diff, index) => {
+    const baseSize = resultsBaseQuery[index].size
+    const localSize = resultsLocalQuery[index].size
+    const sign = diff.percentageChange > 0 ? '+' : ''
+    message += `| ${formatName(diff.name)} | ${baseSize} | ${localSize} | ${sign}${diff.percentageChange}% |\n`
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about formatting the reported size in a more readable format?
ex: `XXXXX` -> `XX.X kB`